test/lib/Chainweb/Test/Pact5/Utils.hs

```diff
@@ -192,8 +201,8 @@ withRunPactService logger v cid pactQueue mempool webBHDb payloadDb pactServiceC
 
 -- | Insert a 'Pact5.Transaction' into the mempool. The mempool currently operates by default on
 --   'Pact4.UnparsedTransaction's, so the txs have to be converted.
-insertMempool :: MempoolBackend Pact4.UnparsedTransaction -> InsertType -> [Pact5.Transaction] -> IO ()
-insertMempool mp insertType txs = do
+mempoolInsert5 :: MempoolBackend Pact4.UnparsedTransaction -> InsertType -> [Pact5.Transaction] -> IO ()
```

Comment:
Maybe `mempoolInsertPact5` is a bit more clear?

Reply:
Yeah that's fair. As long as it starts with `mempoolInsert` rather than `insertMempool`.